src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSerializer.cs

```diff
@@ -275,6 +278,48 @@ private ResourceContext GetResourceContext(object graph, ODataSerializerContext
             }
         }
 
+        internal void WriteDeltaNavigationProperties(SelectExpandNode selectExpandNode, ResourceContext resourceContext, ODataWriter writer)
```

Comment:
Should we also have an async version of this method?

Reply:
is it there. WriteDeltaNavigationPropertiesAsync